src/js/utils/helpers.js

```diff
@@ -14,7 +14,7 @@ import {
 } from 'utils/strings';
 import Timer from 'api/timer';
 import { tryCatch, JwError as Error } from 'utils/trycatch';
-import _ from 'utils/underscore';
+import { indexOf } from './underscore';
```

Comment:
Please keep the import paths. I don't want to mix relative paths with entry (src) relative.